platform_versioned_docs/version-24.2/compute-envs/azure-batch.mdx

```diff
@@ -250,15 +304,83 @@ Create a Batch Forge Azure Batch compute environment:
 See [Launch pipelines](../launch/launchpad.mdx) to start executing workflows in your Azure Batch compute environment.
 :::
 
-## Manual
+### Manual
 
-This section is for users with a pre-configured Azure Batch pool. This requires an existing Azure Batch account with an existing pool.
+It is possible to set up Seqera Platform to use a pre-existing Azure Batch pool. This allows the use of more advanced Azure Batch features, such as custom VM images and private networking.
```

Comment:
Should we link out to MSFT pages that explain the benefits of why you might want these features?